src/textual/design.py

```diff
@@ -190,6 +190,21 @@ def luminosity_range(spread) -> Iterable[tuple[str, float]]:
             colors["text-muted"] = "ansi_default"
             colors["text-disabled"] = "ansi_default"
 
+        # The cursor color for widgets such as OptionList, DataTable, etc.
+        colors["cursor"] = accent.hex
```

Comment:
Note to self - this probably would need a more unique name is it could interact poorly with user defined variables?